test/Microsoft.AspNet.SignalR.Tests/Server/Transports/ForeverTransportFacts.cs

```diff
@@ -94,6 +95,29 @@ public void AbortUrlTriggersConnectionAbort()
             Assert.Equal("c-1", abortedConnectionId);
         }
 
+        [Fact]
+        public void AbortUrlTriggersContentTypeSet()
+        {
+            var request = new Mock<IRequest>();
+            var qs = new NameValueCollection();
+            request.Setup(m => m.QueryString).Returns(new NameValueCollectionWrapper(qs));
+            request.Setup(m => m.LocalPath).Returns("/test/echo/abort");
+
+            var response = new Mock<IResponse>();
+            var counters = new Mock<IPerformanceCounterManager>();
+            var heartBeat = new Mock<ITransportHeartbeat>();
+            var hostContext = new HostContext(request.Object, response.Object);
+            var transportConnection = new Mock<ITransportConnection>();
+            var traceManager = new Mock<ITraceManager>();
+            var transport = new Mock<ForeverTransport>(hostContext, null, heartBeat.Object, counters.Object, traceManager.Object, null)
+            {
+                CallBase = true
+            };
+
+            transport.Object.ProcessRequest(transportConnection.Object).Wait();
+            response.VerifySet(r => r.ContentType = It.IsAny<string>(), "ContentType not set");
```

Comment:
Could you assert the specific content type? We're trying to avoid simple "was this property set" or "was this method called" test cases moving forward since they are brittle and don't test the actual desired outcome.